python-sdk/README.md

```diff
@@ -0,0 +1,135 @@
+# Eppo Python SDK
+
+[![Test and lint SDK](https://github.com/Eppo-exp/rust-sdk/actions/workflows/ci.yml/badge.svg)](https://github.com/Eppo-exp/rust-sdk/actions/workflows/ci.yml)
+
+[Eppo](https://www.geteppo.com/) is a modular flagging and experimentation analysis tool. Eppo's Python SDK is built to make assignments in multi-user server side contexts. Before proceeding you'll need an Eppo account.
+
+## Features
+
+- Feature gates
+- Kill switches
+- Progressive rollouts
+- A/B/n experiments
+- Mutually exclusive experiments (Layers)
+- Dynamic configuration
+
+## Installation
+
+```shell
+pip install eppo-server-sdk
+```
+
+## Quick start
+
+Begin by initializing a singleton instance of Eppo's client. Once initialized, the client can be used to make assignments anywhere in your app.
+
+#### Initialize once
+
+```python
+import eppo_client
+from eppo_client import Config, AssignmentLogger
+
+client_config = Config(api_key="<SDK-KEY-FROM-DASHBOARD>")
+eppo_client.init(client_config)
+```
+
+
+#### Assign anywhere
+
+```python
+import eppo_client
+
+client = eppo_client.get_instance()
+user = get_current_user()
+
+variation = eppoClient.get_boolean_assignment(
+    'show-new-feature',
+    user.id,
+    { 'country': user.country },
+    False
+)
+```
+
+## Assignment functions
+
+Every Eppo flag has a return type that is set once on creation in the dashboard. Once a flag is created, assignments in code should be made using the corresponding typed function:
+
+```python
+get_boolean_assignment(...)
+get_numeric_assignment(...)
+get_integer_assignment(...)
+get_string_assignment(...)
+get_json_assignment(...)
+```
+
+Each function has the same signature, but returns the type in the function name. For booleans use `get_boolean_assignment`, which has the following signature:
+
+```python
+get_boolean_assignment(
+    flag_key: str,
+    subject_key: str,
+    subject_attributes: Dict[str, Any],
+    default_value: bool
+) -> bool:
+  ```
+
+## Initialization options
+
+The `init` function accepts the following optional configuration arguments.
+
+| Option | Type | Description | Default |
+| ------ | ----- | ----- | ----- |
+| **`assignment_logger`**  | AssignmentLogger | A callback that sends each assignment to your data warehouse. Required only for experiment analysis. See [example](#assignment-logger) below. | `None` |
+| **`is_graceful_mode`** | bool | When true, gracefully handles all exceptions within the assignment function and returns the default value. | `True` |
+| **`poll_interval_seconds`** | int | The interval in seconds at which the SDK polls for configuration updates. | `300` |
+| **`poll_jitter_seconds`** | int | The jitter in seconds to add to the poll interval. | `30` |
```

Comment:
does rust do this? Do we pass this through? I wonder if we just drop this. In other SDKs we just do hard-coded 10% polling interval

Reply:
Yes. This configuration is passed through and the core applies the jitter